MovieApp/NetworkService.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  NetworkService.swift
+//  MovieApp
+//
+//  Created by Five on 05.05.2022..
+//
+
+import Foundation
+
+
+class NetworkService {
```

Comment:
As mentioned in another comment, it was intended for NetworkService class to hold all the code responsible for fetching data from the server, parsing it and returning the model objects to callers (in this case view controllers). 

So this class should have several methods such as fetchMovie, fetchMovieList or whatever, and those methods should prepare URLRequests and call this `executeUrlRequest` method, and parse the response of this method to whatever. And then return the result through completion to the view controller.